react-hot-button-component/my-app/src/Button.js

```diff
@@ -0,0 +1,35 @@
+import './HotButton.css';
+import { useState } from 'react';
+
+export default function HotButton(props) {
```

Comment:
For code clarity, you should use destructuring in the props parameter of your React components. This makes it much easier to figure out what props to pass to the component, and means you don't have to dereference `prop` in your own code.

As a reminder, the syntax is
`HotButton({ text })`